scipy/stats/_stats_py.py

```diff
@@ -1805,6 +1805,9 @@ def kurtosistest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
     hypothesis [4]_.
 
     """
+    xp = array_namespace(a)
+    a, axis = _chk_asarray(a, axis, xp=xp)
```

Comment:
For NumPy, `_axis_nan_policy` decorator takes care of raveling for `axis=None`. It doesn't right now for array API, so bring back `_chk_asarray`.